app/assets/docs/001-introduction.md

```diff
@@ -4,7 +4,9 @@ Hello, and welcome to the Peercoin Documentation website. We hope to help you un
 
 ## Peercoin Genesis: 2012
 
-Since the creation of Bitcoin (Nakamoto 2008), proof-of-work has been the predominant design of peer-to-peer crypto currency. The concept of proof-of-work has been the backbone of minting and security model of Nakamoto’s design.
+Since the creation of Bitcoin (Nakamoto 2008), proof-of-work has been the predominant design of peer-to-peer crypto currency. The concept of proof-of-work has been the backbone of mining and security model of Nakamoto’s design.  In 2012, Sunny King and Scott Nadal proposed an alternative form of concensus that was both more secure, but also energy efficient.  Proof-of-work, however is subject to centralized mining, large amounts of power consumption, and majority attacks.  Proof-of-stake sought to provide sustainability and improved security through energy efficient staking and time-based confirmations (coin age).  Instead of electricity and computing power, time could be used as a verification method that would prevent many of the issues plaguing proof-of-work consensus.  Proof-of-stake guards the blockchain, keeping users safe, while dynamic proof-of-work mining provides economic competition and maintains a balance in distribution.  Dynamic mining and staking allow for around 1% inflation a year, making it extremely practical for long term use.
```

Comment:
> Proof-of-work, however is subject to centralized mining,

Too generic. I'd say it's susceptible to miner centralization as consequence of natural process of economies of scale.